src/core/server/app/router/api/account.ts

```diff
@@ -34,6 +35,7 @@ export function createNewAccountRouter(
   router.put("/invite", jsonMiddleware, inviteHandler(app));
 
   router.get("/downloadcheck", downloadCheckHandler(app));
+  router.get("/download", adminDownloadHandler(app));
```

Comment:
I know @wyattjoh had opinions on the `/downloadcheck` endpoint but I'm happy with `/download?token=` for the GET request, so in that context maybe `/downloadcheck` makes more sense?

Reply:
Alternatively, if we have the requesting user from our auth token, maybe we can check if the user asking to download is an admin.

If they are admin, ignore rate limiting, else, rate limit.

Reply:
I was just thinking a POST request doesn't make as much sense for the admin endpoint